kernel/fde/reveal_key.go

```diff
@@ -40,8 +40,10 @@ import (
 type RevealKeyRequest struct {
 	Op string `json:"op"`
 
-	SealedKey []byte `json:"sealed-key,omitempty"`
-	KeyName   string `json:"key-name,omitempty"`
+	SealedKey []byte           `json:"sealed-key,omitempty"`
+	Handle    *json.RawMessage `json:"handle,omitempty"`
+	// depracated for v1
```

Comment:
Typo,
```suggestion
	// deprecated for v1
```

Reply:
Thanks, I think we fix this in the followup